<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandAligner</class>
 <widget class="QMainWindow" name="BandAligner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Band Aligner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/sumbandilasat/icons/toolbar/bandaligner.png</normaloff>:/sumbandilasat/icons/toolbar/bandaligner.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Files</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QRadioButton" name="singleRadioButton">
             <property name="text">
              <string>Single input file</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="multiRadioButton">
             <property name="text">
              <string>Multiple input files</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Bands</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLineEdit" name="bandLineEdit">
                <property name="text">
                 <string>/home/cstallmann/data/ssat/I045C/16bit/merged.tif</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="bandButton">
                <property name="minimumSize">
                 <size>
                  <width>27</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QToolButton" name="addButton">
                <property name="minimumSize">
                 <size>
                  <width>27</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="resources.qrc">
                  <normaloff>:/sumbandilasat/icons/general/add.png</normaloff>:/sumbandilasat/icons/general/add.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>21</width>
                  <height>21</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>40</number>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>40</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>40</number>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>40</number>
             </attribute>
             <column>
              <property name="text">
               <string>Band</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>File</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLineEdit" name="outputLineEdit">
                <property name="text">
                 <string>/home/cstallmann/data/ssat/I045C/16bit/out.tif</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="outputButton">
                <property name="minimumSize">
                 <size>
                  <width>27</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Format</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="outputComboBox">
             <item>
              <property name="text">
               <string>GeoTiff</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Disparity map</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="0">
            <widget class="QWidget" name="disparityWidget" native="true">
             <layout class="QFormLayout" name="formLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>X file</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QGridLayout" name="gridLayout_7">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="xLineEdit"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QToolButton" name="xButton">
                   <property name="minimumSize">
                    <size>
                     <width>27</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>X format</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="xComboBox">
                <item>
                 <property name="text">
                  <string>GeoTiff</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Y file</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QGridLayout" name="gridLayout_9">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QToolButton" name="yButton">
                   <property name="minimumSize">
                    <size>
                     <width>27</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="yLineEdit"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Y format</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="yComboBox">
                <item>
                 <property name="text">
                  <string>GeoTiff</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Save disparity map to file</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Alignment</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Reference band</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="referenceComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Block size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
          <property name="value">
           <number>201</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Process</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Progress</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Next</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/sumbandilasat/icons/general/next.png</normaloff>:/sumbandilasat/icons/general/next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Back</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/sumbandilasat/icons/general/back.png</normaloff>:/sumbandilasat/icons/general/back.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/sumbandilasat/icons/general/stop.png</normaloff>:/sumbandilasat/icons/general/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/sumbandilasat/icons/general/start.png</normaloff>:/sumbandilasat/icons/general/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionZoomIn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/zoomin.png</normaloff>:/icons/georeferencer/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/zoomout.png</normaloff>:/icons/georeferencer/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoomExtent">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/zoomextent.png</normaloff>:/icons/georeferencer/zoomextent.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to extent</string>
   </property>
   <property name="toolTip">
    <string>Zoom to extent</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/pan.png</normaloff>:/icons/georeferencer/pan.png</iconset>
   </property>
   <property name="text">
    <string>Pan image</string>
   </property>
   <property name="toolTip">
    <string>Pan image</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/folder.png</normaloff>:/icons/georeferencer/folder.png</iconset>
   </property>
   <property name="text">
    <string>Scan directory</string>
   </property>
   <property name="toolTip">
    <string>Scan directory</string>
   </property>
  </action>
  <action name="actionAddThumbnail">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/add.png</normaloff>:/icons/georeferencer/add.png</iconset>
   </property>
   <property name="text">
    <string>Add thumbnail</string>
   </property>
   <property name="toolTip">
    <string>Add thumbnail</string>
   </property>
  </action>
  <action name="actionRemoveThumbnail">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/remove.png</normaloff>:/icons/georeferencer/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove thumbnail</string>
   </property>
   <property name="toolTip">
    <string>Remove thumbnail</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/settings.png</normaloff>:/icons/georeferencer/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAlready">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/list.png</normaloff>:/icons/georeferencer/list.png</iconset>
   </property>
   <property name="text">
    <string>Working list</string>
   </property>
   <property name="toolTip">
    <string>Working list</string>
   </property>
  </action>
  <action name="actionCrs">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/projection.png</normaloff>:/icons/georeferencer/projection.png</iconset>
   </property>
   <property name="text">
    <string>Select coordinate reference system</string>
   </property>
   <property name="toolTip">
    <string>Select coordinate reference system</string>
   </property>
  </action>
  <action name="actionSelectPoint">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/selectpoint.png</normaloff>:/icons/georeferencer/selectpoint.png</iconset>
   </property>
   <property name="text">
    <string>Select point</string>
   </property>
   <property name="toolTip">
    <string>Select point</string>
   </property>
  </action>
  <action name="actionAcceptPoint">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/acceptpoint.png</normaloff>:/icons/georeferencer/acceptpoint.png</iconset>
   </property>
   <property name="text">
    <string>Accept point</string>
   </property>
   <property name="toolTip">
    <string>Accept point</string>
   </property>
  </action>
  <action name="actionAddGoogle">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/google.png</normaloff>:/icons/georeferencer/google.png</iconset>
   </property>
   <property name="text">
    <string>Add Google satellite layer</string>
   </property>
   <property name="toolTip">
    <string>Add Google satellite layer</string>
   </property>
  </action>
  <action name="actionNextLayer">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/next.png</normaloff>:/icons/georeferencer/next.png</iconset>
   </property>
   <property name="text">
    <string>Next thumbnail</string>
   </property>
   <property name="toolTip">
    <string>Next thumbnail</string>
   </property>
  </action>
  <action name="actionPreviousLayer">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/previous.png</normaloff>:/icons/georeferencer/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous thumbnail</string>
   </property>
   <property name="toolTip">
    <string>Previous thumbnail</string>
   </property>
  </action>
  <action name="actionSelectTable">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/georeferencer/selecttable.png</normaloff>:/icons/georeferencer/selecttable.png</iconset>
   </property>
   <property name="text">
    <string>Select specific thumbnail from table</string>
   </property>
   <property name="toolTip">
    <string>Select specific thumbnail from table</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
